src/prisma/schema.prisma

```diff
@@ -660,6 +665,7 @@ model User {
   publishToLensDefault          Boolean?                         @default(false)
   primaryWalletId               String?                          @db.Uuid
   primaryWallet                 UserWallet?                      @relation(fields: [primaryWalletId], references: [id], onDelete: SetNull, name: "primaryUserWallet")
+  connectOnboarded              Boolean?                         @default(false)
```

Comment:
i think u'r gonna have issues since valnetin has another PR that adds this field. https://github.com/charmverse/core/pull/312#pullrequestreview-2140915869 maybe its better if he doesn't merge his?

Reply:
This was introduced just to make it work with the changes made in valentin's PR. I will make sure not to include it in the final change @mattcasey

Reply:
Merged mine! Should be good!